### PR TITLE
Update project meeds dependencies

### PR DESCRIPTION
**Pull request by halaya**

### CODE REVIEW

**Review by bourasom — DISMISSED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -59,14 +59,14 @@
...
+    <addon.meeds.tasks.id>meeds-tasks</addon.meeds.tasks.id>
+    <addon.meeds.tasks.version>7.0.x-exo-SNAPSHOT</addon.meeds.tasks.version>
     <addon.exo.web-conferencing.id>exo-web-conferencing</addon.exo.web-conferencing.id>
     <addon.exo.web-conferencing.version>7.0.x-SNAPSHOT</addon.exo.web-conferencing.version>
     <addon.exo.jitsi.id>exo-jitsi</addon.exo.jitsi.id>
     <addon.exo.jitsi.version>7.0.x-SNAPSHOT</addon.exo.jitsi.version>
-    <addon.exo.analytics.id>exo-analytics</addon.exo.analytics.id>
-    <addon.exo.analytics.version>7.0.x-exo-SNAPSHOT</addon.exo.analytics.version>
+    <addon.meeds.analytics.id>meeds-analytics</addon.meeds.analytics.id>
+    <addon.meeds.analytics.version>7.0.x-exo-SNAPSHOT</addon.meeds.analytics.version>
```
Comment (boubaker) [inline pom.xml:69]: Seems not used anymore

```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -59,14 +59,14 @@
     <addon.exo.jcr.version>7.0.x-SNAPSHOT</addon.exo.jcr.version>
     <addon.exo.ecms.id>exo-ecms</addon.exo.ecms.id>
     <addon.exo.ecms.version>7.0.x-SNAPSHOT</addon.exo.ecms.version>
-    <addon.exo.tasks.id>exo-tasks</addon.exo.tasks.id>
-    <addon.exo.tasks.version>7.0.x-exo-SNAPSHOT</addon.exo.tasks.version>
+    <addon.meeds.tasks.id>meeds-tasks</addon.meeds.tasks.id>
+    <addon.meeds.tasks.version>7.0.x-exo-SNAPSHOT</addon.meeds.tasks.version>
```
Comment (boubaker) [inline pom.xml:63]: Seems not used anymore as well
```suggestion
```

File: services/src/main/resources/conf/platform.properties [hunk 3]
```diff
@@ -12,7 +12,8 @@ org.exoplatform.calendar=@addon.exo.calendar.version@
 org.exoplatform.forum=@addon.exo.forum.version@
 org.exoplatform.wiki=@addon.exo.wiki.version@
 org.exoplatform.ecms=@addon.exo.ecms.version@
-org.exoplatform.social=@org.exoplatform.social.version@
+org.exoplatform.social=@io.meeds.distribution.version@
+io.meeds.social=@io.meeds.social.version@
```
Comment (boubaker) [inline services/src/main/resources/conf/platform.properties:16]: Seems missing another property, same in `platform-private-distribution` (missing property).
See it here:
https://github.com/Meeds-io/meeds/blob/a91122d0be9f9c939c460be5f45f3a757463ac4e/services/plf-community-edition-service/src/main/resources/conf/platform.properties#L32

```suggestion
io.meeds.social=@io.meeds.social.version@
io.meeds.distribution=@io.meeds.social.version@
```

**Review by boubaker — DISMISSED** [review]

**Review by hbenali — DISMISSED** [review]

**Pull request merged**